static/js/compose.js

```diff
@@ -404,6 +404,7 @@ export function initialize() {
 
     $("#compose-textarea").on("input propertychange", () => {
         compose_validate.check_overflow_text();
+        compose_validate.warn_if_topic_resolved();
     });
```

Comment:
This has a potential performance issue; we're doing a bunch of work every time content is typed into the compose box. I suppose we could either profile to confirm that it's so fast it doesn't matter, or arrange to avoid doing work on every key (E.g. by checking a global `compose_validate` variable for whether the widget is already displayed)

Reply:
Right, I didn't think of that in the first place! Now, I have defined a new function in `compose_validate` that'd check if the warning widget exists already, which prevents the calling of the `warn_if_topic_resolved` function on every keypress.